docs/v4.0/layers/point-density-grid-layer.md

```diff
@@ -0,0 +1,63 @@
+<p align="right">
+  <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="64-bit" />
+</p>
+
+# PointDensityGridLayer
+
+The PointDensityGridLayer renders a grid heatmap based on an array of points.
+It takes the constant size all each cell, projects points into cells. The color 
+and height of the cell is scaled by number of points it contains.
+
+
+<div align="center">
+  <img height="300" src="/demo/src/static/images/point-density-grid.png" />
```

Comment:
- The gif is cool but pretty big @ 316K demo/src/static/images/point-density-grid.js 
- Also, what's up with the file type - should it have another type (looks like it's called .js above?). 
- General question - Should we consider having smaller fast loading screenshot for the docs, or should we go with gifs for more layers?